src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/ApprenticeshipStoppedInform.cshtml

```diff
@@ -2,6 +2,7 @@
 
 @{
     ViewData["Title"] = "Changing training provider - Apprenticeship stopped";
+    ViewBag.GaData.Vpv = "/change-provider/apprentice-stopped";
```

Comment:
could we make this consistent with almost all the others? `/accounts/apprentices/manage/change-provider/apprentice-stopped`